test/martech/analytics.test.js

```diff
@@ -0,0 +1,21 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+
+describe('Analytics', async () => {
+  beforeEach(async () => {
+    await readFile({ path: './mocks/body.html' });
+    const analytics = await import('../../libs/martech/analytics.js');
+    document.body.outerHTML = await readFile({ path: './mocks/body.html' });
+    document.querySelectorAll('main > div').forEach((section, idx) => analytics.decorateSectionAnalytics(section, idx));
+  });
+  it('should decorate w attributes', async () => {
```

Comment:
w?

Reply:
not sure what you mean

Reply:
I believe the test message, it says 'should decorate w attributes'. typo?

Reply:
Well, usually `w/` translates to `with`; since it's not so clear, maybe replace with either of those

Reply:
updated to "with"